lib/ecs_deploy/instance_fluctuation_manager.rb

```diff
@@ -132,8 +142,26 @@ def extract_target_container_instances(decrease_count, az_to_container_instances
       target_container_instances
     end
 
+    def wait_until_stop_old_tasks(task_arns)
+      @logger.info("All old tasks: #{task_arns.size}")
+      running_tasks = task_arns.each_slice(MAX_DESCRIBABLE_ECS_TASK_COUNT).flat_map do |arns|
+        ecs_client.describe_tasks(cluster: @cluster, tasks: arns).tasks
+      end.select do |task|
+        task.desired_status == "STOPPED" && task.last_status == "RUNNING"
```

Comment:
This line should be `task.desired_status == "RUNNING" || (task.desired_status == "STOPPED" && task.last_status == "RUNNING")`.

Reply:
In this method, `taks.desired_status` must not be `"RUNNING"` because the `stop_tasks` method has stopped all running tasks in a container instance.

Reply:
As far as I can see, `stop_tasks` only stops tasks that don't belong to a service.

Reply:
The tasks that belong to tasks will stop after their container becomes in the status "DARINING". We mustn't stop such tasks explicitly because services with their ALB have to drain connections before stopping.